apps/hash-frontend/src/pages/[shortname].page/profile-bio.tsx

```diff
@@ -41,7 +41,8 @@ export const ProfileBio: FunctionComponent<{
     GetEntityQueryVariables
   >(getEntityQuery, {
     variables: {
-      entityId: profile.hasBio!.profileBioEntity.metadata.recordId.entityId,
+      // eslint-disable-next-line @typescript-eslint/no-non-null-asserted-optional-chain -- potential crash otherwise
+      entityId: profile.hasBio?.profileBioEntity.metadata.recordId.entityId!,
       ...blockCollectionContentsGetEntityVariables,
     },
     skip: !profile.hasBio,
```

Comment:
I think this is happening because the entire arguments object is evaluated before it is passed to the hook, so even though it has no use for the variables if the query is skipped `profile.hasBio` is still being accessed. Will keep this in mind in future, thanks for catching.

Reply:
Ah of course it does, yes. Thanks!